core/src/ast_builder/index_item/primary_key.rs

```diff
@@ -0,0 +1,15 @@
+use {super::PrimaryKeyCmpExprNode, crate::ast_builder::ExprNode};
+
+#[derive(Clone, Debug)]
+pub struct PrimaryKeyNode;
+
+impl<'a> PrimaryKeyNode {
+    pub fn eq<T: Into<ExprNode<'a>>>(self, expr: T) -> PrimaryKeyCmpExprNode<'a> {
+        PrimaryKeyCmpExprNode::new(expr.into())
```

Comment:
It currently goes to `PrimaryKeyCmpExpNode` but could we consider to use `IndexItemNode`?
may be we don't need `PrimaryKeyCmpExprNode` which can be merged into `IndexItemNode`